config/config_defaults.yml

```diff
@@ -61,5 +61,6 @@
 :workflow:
   :url:
   :timeout: 60
+  :logfile: workflow_service.log
```

Comment:
In the context of this client, there is no `./log` directory.  In the context of your **application**, there is.